src/app/main/main.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, ViewChild, ElementRef, Input } from '@angular/core';

```

Comment:
I would like to have iframe code as a separate PR, cause it's not required for project setup and corresponds to different task